builtin/providers/google/resource_google_project.go

```diff
@@ -55,36 +79,57 @@ func resourceGoogleProject() *schema.Resource {
 	}
 }
 
-// This resource supports creation, but not in the traditional sense.
-// A new Google Cloud Project can not be created. Instead, an existing Project
-// is initialized and made available as a Terraform resource.
 func resourceGoogleProjectCreate(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
 
-	project, err := getProject(d, config)
-	if err != nil {
-		return err
-	}
+	pid := d.Get("project_id").(string)
+
+	// See if project already exists. If it doesn't, create it.
+	if _, err := config.clientResourceManager.Projects.Get(pid).Do(); err != nil {
```

Comment:
Fixed (this code path is gone; no longer accounting for project import in the create lifecycle)